solr/core/src/java/org/apache/solr/handler/admin/ZookeeperInfoHandler.java

```diff
@@ -110,6 +112,18 @@ public Category getCategory() {
     none, name, status
   }
 
+  @Override
+  public PermissionNameProvider.Name getPermissionName(AuthorizationContext request) {
+    SolrParams params = request.getParams();
+    String path = params.get("path", "");
+    String detail = params.get("detail", "false");
```

Comment:
Can we add a field constant for this param (and perhaps others) to more clearly show we depend on this name in the code from two places?  Generally I find string constants to be an over-used coding habit yet ironically I propose it here because I think it would decrease a security risk if the code evolves.  I don't feel strongly; feel free to ignore.  Maybe this could be ameliorated with a simple comment in handleRequestBody saying simply `//note: these params are used in getPermissionName at runtime`

Reply:
Done